templates/firefox-ios.xliff

```diff
@@ -3707,6 +3731,279 @@ Private Tab</source>
+        <source>Reliable reviews. We believe the reviews are likely from real customers who left honest, unbiased reviews.</source>
+        <note>Reliable reviews label from How we determine review quality card displayed in the shopping review quality bottom sheet.</note>
+      </trans-unit>
+      <trans-unit id="Shopping.ReviewQualityCard.SubHeadline.Label.v120" xml:space="preserve">
+        <source>We assign each product’s reviews a *letter grade* from A to F.</source>
+        <note>Label of the sub headline from How we determine review quality card displayed in the shopping review quality bottom sheet. The *text inside asterisks* denotes part of the string to bold, please leave the text inside the '*' so that it is bolded correctly.</note>
+      </trans-unit>
+      <trans-unit id="Shopping.ReviewQualityCard.UnreliableReviews.Label.v120" xml:space="preserve">
+        <source>Unreliable reviews. We believe the reviews are likely fake or from biased reviewers.</source>
+        <note>Unreliable reviews label from How we determine review quality card displayed in the shopping review quality bottom sheet.</note>
+      </trans-unit>
+      <trans-unit id="Shopping.SettingsCard.Collapsed.AccessibilityLabel.v120" xml:space="preserve">
+        <source>Settings Card Collapsed</source>
+        <note>Accessibility label for the up chevron, from Settings Card View displayed in the shopping review quality bottom sheet.</note>
+      </trans-unit>
+      <trans-unit id="Shopping.SettingsCard.Expanded.AccessibilityLabel.v120" xml:space="preserve">
+        <source>Settings Card Expanded</source>
+        <note>Accessibility label for the down chevron, from Settings Card View displayed in the shopping review quality bottom sheet.</note>
+      </trans-unit>
+      <trans-unit id="Shopping.SettingsCard.Footer.Action.v120" xml:space="preserve">
+        <source>Review checker is powered by %1$@ by %2$@</source>
+        <note>Action title of the footer underneath the Settings Card displayed in the shopping review quality bottom sheet. The first parameter will be replaced by the Fakespot app name and the second parameter by the company name of Mozilla.</note>
+      </trans-unit>
+      <trans-unit id="Shopping.SettingsCard.Label.Title.v120" xml:space="preserve">
+        <source>Settings</source>
+        <note>Title of the settings card displayed in the shopping review quality bottom sheet.</note>
+      </trans-unit>
+      <trans-unit id="Shopping.SettingsCard.RecommendedProducts.Label.v120" xml:space="preserve">
+        <source>Show products recommended by %@</source>
+        <note>Label of the switch from settings card displayed in the shopping review quality bottom sheet. The placeholder will be replaced with the app name.</note>
+      </trans-unit>
+      <trans-unit id="Shopping.SettingsCard.TurnOff.Button.v120" xml:space="preserve">
+        <source>Turn Off Review Checker</source>
```

Comment:
want to double check that title case is required here (I think it may?)

Reply:
Fixed in https://github.com/mozilla-mobile/firefox-ios/pull/16940